tests/build.rs

```diff
@@ -136,6 +136,73 @@ fn it_builds_with_webpack_wast() {
     cleanup(fixture);
 }
 
+#[test]
+fn it_builds_with_kv_metadata_webpack() {
+    let fixture = "kv_metadata_webpack";
+    create_temporary_copy(fixture);
+
+    settings! {fixture, r#"
+        type = "webpack"
+
+        [[kv-namespaces]]
+        binding = "bind"
+        id = "id"
+    "#};
+
+    build(fixture);
+    assert!(fixture_out_path(fixture).join("metadata.json").exists());
+
+    let metadata = fs::read_to_string(fixture_out_path(fixture).join("metadata.json"))
+        .expect("could not read metadata");
+    assert_eq!(metadata, r#"{"body_part":"script","bindings":[{"type":"kv_namespace","name":"bind","namespace_id":"id"}]}"#);
+    cleanup(fixture);
+}
+
+#[test]
+fn it_builds_with_kv_metadata_js() {
+    let fixture = "kv_metadata_js";
+    create_temporary_copy(fixture);
+
+    settings! {fixture, r#"
+        type = "javascript"
+
+        [[kv-namespaces]]
+        binding = "bind"
+        id = "id"
+    "#};
+
+    build(fixture);
+    assert!(fixture_out_path(fixture).join("metadata.json").exists());
+
+    let metadata = fs::read_to_string(fixture_out_path(fixture).join("metadata.json"))
+        .expect("could not read metadata");
+    assert_eq!(metadata, r#"{"body_part":"script","bindings":[{"type":"kv_namespace","name":"bind","namespace_id":"id"}]}"#);
+    cleanup(fixture);
+}
+
+#[cfg(feature = "slow_tests")]
+#[test]
+fn it_builds_with_kv_metadata_rust() {
```

Comment:
is there also a test for `it_builds_with_wasm_bindings_rust`? because reading the code it looks like maybe no?

Reply:
Yes I agree, we don't yet.